index.html

```diff
@@ -115,6 +115,36 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ Lord-x7 card START ________  -->
+        <div class="card">
+          <p class="name">Lord-x7</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/Parihar__07" target="_blank">Parihar07</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/harash-parihar-874785225" target="_blank">Parihar07</a>
+            <i class="fab fa-discord"></i>
+            <a href="https://discord.gg/discord.app/users/Lordx7#0394" target="_blank">Parihar07</a>
+          </p>
+          <p class="about">I'm a student from India .Currently focusing On Data Structures and Algorithms,Linux and Python develeopment</p>
+          <div class="resources">
+            <p>3 Useful Resources: 
+            </p>
+            <ul>
+              <li>
+                <a href="www.linkedin.com" target="_blank" title="First Resource">Linkedin</a>
+              </li>
+              <li>
+                <a href="https://www.google.com/search?q=florian+dedov+python+bible&rlz=1C1CHBF_enIN1003IN1003&oq=florian+dedov+python+bible&aqs=chrome..69i57.10361j0j1&sourceid=chrome&ie=UTF-8&si=AC1wQDAtx2BbmyBmykHcHVhKaK47clF_XdL8sYdO8Trpk4O-6hko1BlMx51g9LBviEWcmLSfS3xRLaHkRtdX167nKuNbwXZS4v8z6AfcaQMIL_cdeyewnDkmUd5WSHlN2WcpfDqErcT7qVU5da3RDQWZBlfQ-h2GuLxOW-fV-Na_vlC7ryJXmfizHNbvCLQncNrCFNi0a_vFgKV94qYlRlboCVwNLRCl3HD2l3WMAmKSbsMiEtxJI8Qvt1GkDE6oE9OzgVWv8veLZl0kO0epJbnsuP7ehc2ZLgGhdI0ovtD5NzsxMW92JIA%3D&ictx=1&ved=2ahUKEwij_Krrlqb7AhWCSWwGHbZaBQIQnZMFegQIZBAC" target="_blank" title="Second Resource">Book to learn Python</a>
```

Comment:
Remove the google search and put an actual link that may lead to the book itself, ex: author website/pdf download link etc.

Reply:
Done